pkg/blobserver/gethandler/get.go

```diff
@@ -33,6 +33,11 @@ import (
 	"go4.org/readerutil"
 )
 
+const (
+	// TODO: Expose this to the config somehow?
```

Comment:
Do we need that todo? Why would one want to configure this?

Reply:
```suggestion
```